lib/state/actions/home_page_actions.dart

```diff
@@ -13,6 +14,26 @@ class GetPokemonAction extends LoadingAction {
   @override
   Future<AppState> reduce() async {
     final pokemon = await PokemonHandler.getPokemon();
-    return state.copyWith(pokemon: pokemon);
+    return state.copyWith(
+      pokemon: pokemon,
+      isVisible: false,
```

Comment:
Create separate action for updating the `isVisible` property in state. Then just call it in `before() `or `after() `method of this action

Reply:
```
@override
  void after() {
    dispatch(ChangeFilterCancelVisibilityAction(isShown: false));
    super.after();
  }
```